profiler/src/main/proto/profile.proto

```diff
@@ -0,0 +1,212 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Should a link to the original pprof proto file(s) be included?

Reply:
If this ends up used then it will need to go through gdi spec where it will be documented.